### PR TITLE
reporter-web-app: Increase the Yarn network timeout

### LINKED ISSUE #2173: Docker build fails on Windows 10
Docker engine version: 19.03.5
Docker Desktop version: 2.1.0.5

I'm trying to build the project using Docker as described in the README, i.e. by running `docker build -t ort .` in the source directory with a Docker daemon running in the background. I'm on 
Windows 10 (unfortunately) and I have tried to build with Git Bash and Powershell.

The build fails with the following error:

````
> Task :reporter-web-app:yarnBuild   [16/212]
/root/.gradle/nodejs/node-v10.15.3-linux-x64/bin/node: /usr/lib/libstdc++.so.6: no version information available (required by /root/.gradle/nodejs/node-v10.15.$
-linux-x64/bin/node)
/root/.gradle/nodejs/node-v10.15.3-linux-x64/bin/node: /usr/lib/libstdc++.so.6: no version information available (required by /root/.gradle/nodejs/node-v10.15.$
-linux-x64/bin/node)
/root/.gradle/nodejs/node-v10.15.3-linux-x64/bin/node: /usr/lib/libstdc++.so.6: no version information available (required by /root/.gradle/nodejs/node-v10.15.$
-linux-x64/bin/node)
/root/.gradle/nodejs/node-v10.15.3-linux-x64/bin/node: /usr/lib/libstdc++.so.6: no version information available (required by /root/.gradle/nodejs/node-v10.15.$
-linux-x64/bin/node)
/root/.gradle/nodejs/node-v10.15.3-linux-x64/bin/node: /usr/lib/libstdc++.so.6: no version information available (required by /root/.gradle/nodejs/node-v10.15.$
-linux-x64/bin/node)
/root/.gradle/nodejs/node-v10.15.3-linux-x64/bin/node: /usr/lib/libstdc++.so.6: no version information available (required by /root/.gradle/nodejs/node-v10.15.$
-linux-x64/bin/node)
yarn run v1.21.1
$ node scripts/build.js
/root/.gradle/nodejs/node-v10.15.3-linux-x64/bin/node: /usr/lib/libstdc++.so.6: no version information available (required by /root/.gradle/nodejs/node-v10.15.$
-linux-x64/bin/node)
/root/.gradle/nodejs/node-v10.15.3-linux-x64/bin/node: /usr/lib/libstdc++.so.6: no version information available (required by /root/.gradle/nodejs/node-v10.15.$
-linux-x64/bin/node)
/root/.gradle/nodejs/node-v10.15.3-linux-x64/bin/node: /usr/lib/libstdc++.so.6: no version information available (required by /root/.gradle/nodejs/node-v10.15.$
-linux-x64/bin/node)
/root/.gradle/nodejs/node-v10.15.3-linux-x64/bin/node: /usr/lib/libstdc++.so.6: no version information available (required by /root/.gradle/nodejs/node-v10.15.$
-linux-x64/bin/node)
/root/.gradle/nodejs/node-v10.15.3-linux-x64/bin/node: /usr/lib/libstdc++.so.6: no version information available (required by /root/.gradle/nodejs/node-v10.15.$
-linux-x64/bin/node)
/root/.gradle/nodejs/node-v10.15.3-linux-x64/bin/node: /usr/lib/libstdc++.so.6: no version information available (required by /root/.gradle/nodejs/node-v10.15.$
-linux-x64/bin/node)
Creating an optimized production build...
Browserslist: caniuse-lite is outdated. Please run next command `yarn upgrade caniuse-lite browserslist`
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with signal "SIGKILL".

> Task :reporter-web-app:yarnBuild FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':reporter-web-app:yarnBuild'.
> Process 'command '/root/.gradle/nodejs/node-v10.15.3-linux-x64/bin/node'' finished with non-zero exit value 1

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.


* Get more help at https://help.gradle.org

Deprecated Gradle features were used in this build, making it incompatible with Gradle 7.0.BUILD FAILED in 18m 27s

Use '--warning-mode all' to show the individual deprecation warnings.
See https://docs.gradle.org/6.1.1/userguide/command_line_interface.html#sec:command_line_warnings
45 actionable tasks: 45 executed
The command '/bin/sh -c scripts/set_gradle_proxy.sh &&     ./gradlew --no-daemon :cli:distTar' returned a non-zero code: 1
````

I can provide the full build log if needed.

Reading the comments on #2084, it seems Docker support on Windows is relatively new and using ORT with Docker on Windows is discouraged, but I would still apprecaite any support.

Thank you.
Uwe

### PR DESCRIPTION
**Pull request by sschuberth**
The default network timeout of 30 seconds seems to be too low in some
cases, so increase it to 5 minutes which seems like a good compromise
between slow networks and not blocking the build forever, also see [1].

Resolves #2173.

[1] yarnpkg/yarn#4890 (comment)

Signed-off-by: Sebastian Schuberth <sebastian.schuberth@bosch.io>

### CODE REVIEW

File: reporter-web-app/.yarnrc [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (fviernau) [inline reporter-web-app/.yarnrc]: commit-msg: Would it make sense to provide a rationale for choosing`10 minutes` as value?

Comment (sschuberth) [reply reporter-web-app/.yarnrc]: Please see the link in the commit message; TL;DR it's arbitrarily high.

Comment (fviernau) [reply reporter-web-app/.yarnrc]: I read the link and decided to stop at 'Here I set it to 10 minutes for OVERKILL'.

Comment (sschuberth) [reply reporter-web-app/.yarnrc]: > I read the link and decided to stop at 'Here I set it to 10 minutes for OVERKILL'.

I didn't read any further either :laughing:

Comment (sschuberth) [reply reporter-web-app/.yarnrc]: But I'll amend the commit message...

Comment (sschuberth) [reply reporter-web-app/.yarnrc]: I decided that at least mentioning the default timeout makes sense, and added that to the commit message, too.

**Pull request merged**